SpaceEnvironment_TotalLaunches/js/area.js

```diff
@@ -0,0 +1,82 @@
+$(function() {
+  $('#hcContainer').highcharts({
+    // Load Data in from Google Sheets
+    data: {
+      googleSpreadsheetKey: '1FpDY2RRA1EW5Pk7cK9tRSstaH6P0f-2GEDc_VjNRI4o',
+      googleSpreadsheetWorksheet: 1,
+      complete: function (data) {
+        // Display legend alphabetically
+        let dataItems = data.series
+        let dataNames = []
+
+          dataItems.forEach(item => {
+              item.legendIndex = ''
+              dataNames.push(item.name)
+            })
+
+            notSorting = dataNames.filter(arrayItem => arrayItem === 'Others')
```

Comment:
This is well done! But you need to define the `notSorting` and `alphaSorting` variables by adding `let`, `var`, or `const` in front of them.